### ISSUE TITLE
Folks would love the Visio files, if possible?

### ISSUE THREAD

**Issue by hilde-msft** [MicrosoftDocs/memdocs#3857]
A GREAT doc - Folks would love the Visio files, if possible?

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: d69ce384-b7cd-58b6-df63-999c851e3423
* Version Independent ID: 4a48ae1f-4572-4da1-0894-6f30f3e84cc4
* Content: [High-level architecture for Microsoft Intune](https://learn.microsoft.com/en-us/mem/intune/fundamentals/high-level-architecture)
* Content Source: [memdocs/intune/fundamentals/high-level-architecture.md](https://github.com/MicrosoftDocs/memdocs/blob/main/memdocs/intune/fundamentals/high-level-architecture.md)
* Service: **microsoft-intune**
* Sub-service: **fundamentals**
* Technology: ****
* GitHub Login: @Erikre
* Microsoft Alias: **erikre**

**Comment by yogkumgit** [comment]
@Erikre Please help us in resolving this issue. Thanks

**Comment by djval34** [comment]
Yes, great doc.  Is it possible to print this as a poster (3' x 2', 4' x 3') so i can post it in my lab?

**Comment by pl4nty** [comment]
+1, I'd like to add docs links for the respective services, similarly to https://intunemaps.com
